SECURITY_ADVISORS

```diff
@@ -0,0 +1,6 @@
+# containerd project security advisors
+#
+# See GOVERNANCE.md for description of role
+#
+# SECURITY ADVISORS
+# GitHub ID, Name, Email address
```

Comment:
Should we have a column for who these advisors may represent?  i.e. a company?

Reply:
Need the member to turn on 2FA for GitHub.

Reply:
I think we should require the member to turn on 2FA for GitHub account, for security reason

Reply:
Good question; guess it depends; if these people represent a company does that also (implicitly) means that if they would move companies, that another person should take their place?

Reply:
I was thinking about this. The email address here should really use a company email. This email will get linked to `security@containerd.io`, so moving jobs should mean updating here. We can leave it up to maintainer discretion. We should avoid adding other mailing lists here though since the goal is to get an individual's advice.

Adding company is good though so those within a company may know who to contact related to containerd security.

Reply:
> if these people represent a company does that also (implicitly) means that if they would move companies, that another person should take their place?

> We can leave it up to maintainer discretion.

+1